doc/LABELS.md

```diff
@@ -9,18 +9,18 @@ Before contributing to the organization, one should be aware of these and keep t
 For keeping the issues in a systematic way, we use labels which describe the type of issue, the  ` programming language ` used in the issue and so on.
 Some of the most used labels are:-
 * ` help-wanted ` which indicates the issue requires help by anyone willing to contribute.
-* ` first-timers-only ` which are meant to welcome newcomers in the community. They need to be well-formatted using the *First-timers_Issue_Template*.
+* ` first-timers-only ` which are meant to welcome newcomers in the community. They need to be well-formatted using the *First-timers_Issue_Template*. :tada: :smiley:
 * ` fto-candidate ` issues are issues which are meant to be solved by **first timers** but they aren't well-formatted. These issues can be converted into ` first-timers-only ` issues using the friendly template.
 * ` bug ` which tells that the issue is regarding one of our programs which faces problems when a certain task is executed. :bug:
-* ` enhancement ` explains that the issue is to improve upon one of our existing features.
-* ` planning ` - These issues can be used as a place for discussion on a long term or a big project.
+* ` enhancement ` explains that the issue is to improve upon one of our existing features. :bulb:
+* ` planning ` - These issues can be used as a place for discussion on a long term or a big project. :computer:
 * ` break-me-up ` says that this certain issue could be and should be broken into smaller self-contained projects for cleaner code separation, more discrete tests, and, easier and iterative collaboration.
-* ` more-detail-please ` tells the issue lacks proper description and perhaps needs code links or the location of the problem.
+* ` more-detail-please ` tells the issue lacks proper description and perhaps needs code links or the location of the problem. :link:
 * Labels like ` HTML ` , ` CSS ` , ` Ruby ` and ` JavaScript ` tell the **programming language** of the issue.
 * ` design ` - This says that the issue requires more design work and discussion (i.e. mockups and sketches).
 * ` documentation ` - This tells that a certain feature lacks proper documentation or needs more documents. :orange_book:
 * ` testing ` - These issues are usually for adding `unit tests`, `integration tests` or any other tests for a particular feature/program. :heavy_check_mark:
-* ` outreach ` - The outreach issues involve community involvement and helping people who're stuck somewhere.
+ * ` outreach ` - The outreach issues involve community involvement and helping people who're stuck somewhere. :tada: :raising_hand_man:
```

Comment:
@bajajvinamr could you remove the extra space before `*`?

Reply:
committed the change